ideascube/monitoring/templates/monitoring/loan.html

```diff
@@ -28,7 +28,7 @@
         </tr>
         {% for loan in loans %}
             <tr>
-                <td>{{ loan.specimen.item }} - {{ loan.specimen.barcode }}</td>
+                <td>{{ loan.specimen.item }} - {{ loan.specimen.pretty_identifier }}</td>
```

Comment:
I really think it would be better to display both the `barcode` and the `serial` here, just like in #659.